packages/admin-ui-docs/docs/documentation/introduction/component-status.mdx

```diff
@@ -8,11 +8,13 @@ sidebar_position: 2
 import { Text, color as getColor } from '@vtex/admin-ui'
 import { StatusTable, componentsStatus } from '@site/src/components/StatusTable'
 
-All of Admin UI component status:
+Last updated: Jan 13, 2023
+
+- <Text csx={{ color: getColor('gray50') }}>Proposed</Text> – Example not yet validated by the Admin UI team that was contributed. The design is in the Team Components library and the code, when it exists, is in the shared Code Sandbox team (WIP). Teams should use these examples for inspiration.
```

Comment:
We already have a [doc section](https://admin-ui-docs-klm6vly9v-admin-team.vercel.app/next/documentation/introduction/contributing#ways-of-contributing-to-admin-ui) saying how to contribute to admin-ui, and in this section, we've also documented the Codesandbox workspace

```suggestion
- <Text csx={{ color: getColor('gray50') }}>Proposed</Text> – Example not yet validated by the Admin UI team that was contributed. The design is in the Team Components library and the code, when it exists, is in the shared [Codesandbox workspace](https://codesandbox.io/dashboard/sandboxes/examples). Teams should use these examples for inspiration.
```